examples/gno.land/r/sys/params/params.gno

```diff
@@ -19,6 +19,7 @@ package params
 
 import (
 	"std"
+	prms "sys/params"
```

Comment:
seems like it doesn't follow what we said in various discussions: #1475 

cc @thehowl @leohhhn

Reply:
Sure, we can adapt to the new naming system after it is finalized and update them all together. 

Currently, this is intentionally defined outside of std because SetPrefixedString() differs from std.SetParamString().
 
std.SetParamString(k, v) is used for arbitrary parameters that can be called from any realm. It may or may not affect system behavior. It depends on whether we have code to capture these arbitrary parameters and build logic around them.

sys.params.SetPrefixedString(module, k, v) is used for module parameters and can only be called from the gno.land/r/sys/params realm. It modifies an existing parameter predefined in the module and will definitely change system behavior.

I am working on adding more documentation to this.

Reply:
updated the comments and doc.go params keeper.

Reply:
@thehowl, what do you think?